src/processout/processout.ts

```diff
@@ -1253,8 +1294,13 @@ module ProcessOut {
                                     success: (data: any) => void,
                                     error: (err: Exception) => void): void {
 
+            // returns this.hppInitialURL only once during the first call from HPP, then returns the endpoint
+            const getEndpoint = (): string => {
```

Comment:
It would be good to improve readability of this function:
```
const getEndpoint = () => {
  if (this.threeDSInitiationRequested) {
    this.threeDSInitationRequested = false;
    return this.threeDSInitiationURL
  }

  return endpoint;
}
```